packages/astro/src/server/build-clerk-hotload-script.ts

```diff
@@ -51,10 +16,10 @@ function buildClerkHotloadScript(locals: APIContext['locals']) {
     publishableKey,
   });
   return `
-  <script src="${scriptSrc}" 
-  data-clerk-script 
-  async 
-  crossOrigin='anonymous' 
+  <script src="${scriptSrc}"
+  data-clerk-js-script
```

Comment:
In Nextjs, we are using `data-clerk-js-script` attribute and `loadClerkJsScript` checks for it. Updated the Astro package to use it as well to have consistency